Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -12,7 +12,22 @@ open class ChatChannelListVC<ExtraData: UIExtraDataTypes>: UIViewController,
     
     public var controller: _ChatChannelListController<ExtraData>!
     public var uiConfig: UIConfig<ExtraData> = .default
-    public var didSelectChannel: (_ChatChannel<ExtraData>) -> Void = { _ in }
+    public lazy var router: ChatChannelListRouter<ExtraData> = .init(rootViewController: self)
```

Comment:
Is it a good idea to have a `NavigationConfig` with router types?

Reply:
Just 1 config is enough :) It should contain all classes we need but you can create a `Navigation` struct within it.

Reply:
I don't think the router should be owned by the VC but I guess it's OK for now.